index.js

```diff
@@ -130,6 +157,14 @@ module.exports = options => {
       return Promise.resolve(null)
     }
 
+    const commandWithoutSemi = command.endsWith(';') ? command.substring(0, command.length - 1) : command
+    if (Object.keys(specialCommands).indexOf(commandWithoutSemi) !== -1) {
+      return evalSpecialSol(commandWithoutSemi, commands)
+        .then(result => {
+          return JSON.stringify(result)
```

Comment:
I would suggest changing this to `JSON.stringify(result, null, 2)` to pretty print the result.